services/ui-src/src/utils/email/email.ts

```diff
@@ -0,0 +1,13 @@
+export const createEmailLink = (emailData: Props): string => {
+  const { address, subject, body } = emailData;
+  const mailtoSubject = subject ? `?${encodeURIComponent(subject)}` : "";
+  const mailtoBody = body ? `&${encodeURIComponent(body)}` : "";
+
+  return `mailto:${address}${mailtoSubject}${mailtoBody}`;
+};
+
+interface Props {
+  address: string;
+  subject?: string;
+  body?: string;
```

Comment:
I talked with design and it sounds like they are finalizing the subjects, but there will be no bodies for now. Could take out until we need it.